xbmc/addons/kodi-dev-kit/include/kodi/gui/General.h

```diff
@@ -170,6 +191,107 @@ inline kodi::HardwareContext GetHWContext()
 }
 //------------------------------------------------------------------------------
 
+//==============================================================================
+/// @ingroup cpp_kodi_gui_general
+/// @brief Get the whitelist resolutions.
+///
+/// @return The whitelist resolutions
+///
+///
+///-------------------------------------------------------------------------
+///
+/// **Example:**
+/// ~~~~~~~~~~~~~{.cpp}
+/// ..
+/// std::vector<SResolutionEntry> resList = kodi::gui::GetWhitelistResolutions();
+/// ..
+/// ~~~~~~~~~~~~~
+///
+inline std::vector<SResolutionEntry> GetWhitelistResolutions()
+{
+  using namespace ::kodi::addon;
+  std::vector<SResolutionEntry> resList;
+  int* widthList{nullptr};
+  int* heightList{nullptr};
+  float* refreshRateList{nullptr};
+  unsigned int size{0};
```

Comment:
About the planned sandbox is would always better to use for any size values a `size_t` as easier to identify from scripts.

Reply:
done